src/Compiler/Xsc.cpp

```diff
@@ -34,7 +34,7 @@ XSC_EXPORT bool CompileShader(
         reflectionData,
         &timePoints
     );
-
+    #if 0 //no sort
```

Comment:
Is there a reason this has been disabled? It was originally used for quick command-line feedback of the code reflection, where sorting can be helpful.

Reply:
As I said all reflection code is copied from @Gabriele91, I know that they need a bit of refinement, I am currently working on normal mapping in my engine when I finish it, I will modify the reflection code and fix the texture binding slot.